packages/control-input/src/Radio.tsx

```diff
@@ -38,9 +39,24 @@ export const Radio = forwardRef<HTMLInputElement, RadioProps>(function Radio(
 	{ children, disabled, invalid, valid, size = 'md', ...props },
 	ref
 ) {
+	const { groupMessageId } = useControlGroupContext();
+	const describedByIds = [
+		groupMessageId || null,
+		props['aria-describedby'] || null,
+	].filter(Boolean);
+
```

Comment:
```suggestion
	const { groupMessageId } = useControlGroupContext();
	const describedByIds = [
		groupMessageId || null,
		props['aria-describedby'] || null,
	].filter(Boolean);
```

I think we could probably just move this into `ControlInput` to avoid the duplication `Radio` and `Checkbox`